collectors/schema.yaml

```diff
@@ -0,0 +1,210 @@
+type: "object"
+properties:
+  name:
+    type: "string"
+  title:
+    type: "string"
+  duplicate_for_virtual_integrations:
+    type: "object"
+    properties:
+      list:
+        type: "array"
+        items:
+          type: "string"
+  overview:
+    type: "object"
+    properties:
+      how_the_collector_works:
```

Comment:
I think we can call it `data_collection` because it describes how the data is collected (and probably briefly what data is collected).